src/E1_coldflow/E1_coldflow.cpp

```diff
@@ -140,10 +159,32 @@ void loop() {
 
     #if SERIAL_INPUT != 1
         RFSerial.println(packet);
-      #endif
+    #endif
     write_to_SD(packet.c_str(), file_name);
+
+      // After getting new pressure data, check injector pressures to detect end of flow:
+  if (sensor->id==1 && Automation::inFlow()){
+    // Serial.println("flowing!");
+    float loxInjector = farrbconvert.sensorReadings[2];
+    float propInjector = farrbconvert.sensorReadings[3];
+    
+    
+
+    if (Solenoids::getLox5()) { //if LOX Main valve open
+      Serial.print("0: "); Serial.println(loxInjector);
+      Automation::detectPeak(loxInjector, 0);
+    }
+    if (Solenoids::getProp5()) { //if Prop Main valve open
+      Serial.print("1: "); Serial.println(propInjector);
+      Automation::detectPeak(propInjector, 1);
+    }
```

Comment:
This looks like it should be its own func in automation

Reply:
moved logic to Automation::detectPeaks()